src/platforms/linux.ts

```diff
@@ -75,6 +75,10 @@ export default class LinuxPlatform implements Platform {
     }
   }
 
+  async deleteProtectedFile(filepath: string) {
+    await run(`sudo rm ${filepath}"`);
```

Comment:
I think a double quote was missed here

Reply:
I super don't know how that happened. It passed all tests--I must have typoed it before commit.